campaigns/views.py

```diff
@@ -65,6 +69,10 @@ def delete(self, response: Response, campaign_id: str):
         except ValidationError as err:
             return Response({"error": err}, status.HTTP_422_UNPROCESSABLE_ENTITY)
 
+
+        if(find_campaign.collected > 0):
+            return Response({"error": "Collected field has to be 0 or lower"}, status.HTTP_403_FORBIDDEN)
```

Comment:
it cannot be lower that 0, we should never have a negative value in the collected field